cpp/common/src/codingstandards/cpp/Compatible.qll

```diff
@@ -0,0 +1,26 @@
+import cpp
+
+predicate typesCompatible(Type t1, Type t2) {
```

Comment:
M3-2-1 uses `areCompatible` for this, but it's a barebones predicate. Can we combine these two predicates together to share across both sets of queries?

Reply:
see comment above for why not